src/test/java/com/michelin/ns4kafka/services/executors/TopicAsyncExecutorTest.java

```diff
@@ -96,18 +96,21 @@ void shouldDeleteTagsAndNotCreateIfEmpty() {
 
         topicAsyncExecutor.alterTags(ns4kafkaTopics, brokerTopics);
 
-        verify(schemaRegistryClient).deleteTag(LOCAL_CLUSTER, CLUSTER_ID_TEST + ":" + TOPIC_NAME, TAG2);
-        verify(schemaRegistryClient).deleteTag(LOCAL_CLUSTER, CLUSTER_ID_TEST + ":" + TOPIC_NAME, TAG3);
+        verify(schemaRegistryClient).dissociateTag(LOCAL_CLUSTER, CLUSTER_ID_TEST + ":" + TOPIC_NAME, TAG2);
+        verify(schemaRegistryClient).dissociateTag(LOCAL_CLUSTER, CLUSTER_ID_TEST + ":" + TOPIC_NAME, TAG3);
     }
 
     @Test
     void shouldCreateTags() {
         Properties properties = new Properties();
         properties.put(CLUSTER_ID, CLUSTER_ID_TEST);
 
-        when(schemaRegistryClient.addTags(anyString(), anyList()))
-            .thenReturn(Mono.empty())
-            .thenReturn(Mono.error(new Exception("error")));
+        when(schemaRegistryClient.associateTags(anyString(), anyList()))
+                .thenReturn(Mono.empty())
+                .thenReturn(Mono.error(new Exception("error")));
```

Comment:
There're issues on this test:

- Remove the 2nd `.thenReturn` and the first one should return `Mono.just(List.of())`

Reply:
take in account